aqua/ssh-askpass-mac/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               xcode 1.0
+
+name                    ssh-askpass-mac
+github.setup            lukas-zronek ${name} 1.2 v
```

Comment:
Using `${name}` in this way is not necessary.

`github.setup` implicity sets `${name}`, among other variables.

So you can delete the `${name}` line, and simply do:
```
github.setup            lukas-zronek ssh-askpass-mac 1.2 v
```

`${name}` will be set as expected.

Reply:
Will do!